src/Symfony/Component/Notifier/Bridge/Twilio/TwilioTransport.php

```diff
@@ -88,7 +88,7 @@ protected function doSend(MessageInterface $message): SentMessage
         if (201 !== $statusCode) {
             $error = $response->toArray(false);
 
-            throw new TransportException('Unable to send the SMS: '.$error['message'].\sprintf(' (see %s).', $error['more_info']), $response);
+            throw new TransportException('Unable to send the SMS: '.$error['message'].\sprintf(' (see "%s").', $error['more_info']), $response);
```

Comment:
this should should be reverted. That's a case where the fabbot rule has a false positive.